packages/helpers/qrcode-modal/src/browser/languages/de.ts

```diff
@@ -1,8 +1,17 @@
-export default {
+import { TextMap } from "../types";
+
+const de: TextMap = {
   choose_preferred_wallet: "Wähle bevorzugte Wallet",
   connect_mobile_wallet: "Verbinde mit Mobile Wallet",
   scan_qrcode_with_wallet: "Scanne den QR-code mit einer WalletConnect kompatiblen Wallet",
-  return_to_mobile_options: "Zurück zur Wallet-Anbieter Übersicht",
-  view_qrcode_option: "QR-code Anzeigen",
   connect: "Verbinden",
+  // TODO: review following translations
```

Comment:
Could you take a look at the following translations? cc @MAkzent

Reply:
looks good!